solr/bin/solr.cmd

```diff
@@ -1187,12 +1187,12 @@ IF "%ENABLE_REMOTE_JMX_OPTS%"=="true" (
   set REMOTE_JMX_OPTS=
 )
 
-REM Enable java security manager (limiting filesystem access and other things)
-IF "%SOLR_SECURITY_MANAGER_ENABLED%"=="true" (
-  set SECURITY_MANAGER_OPTS=-Djava.security.manager ^
--Djava.security.policy="%SOLR_SERVER_DIR%\etc\security.policy" ^
--Djava.security.properties="%SOLR_SERVER_DIR%\etc\security.properties" ^
--Dsolr.internal.network.permission=*
+REM Enable java security manager by default (limiting filesystem access and other things)
+IF NOT DEFINED SOLR_SECURITY_MANAGER_ENABLED (
```

Comment:
to keep the `.cmd` logic simple, we may just want to set the environment variable, if it is not already set. It limits the changes to what we want to do (changing the default) with just a simple if-statement. And we can avoid wrestling with any of the other parts of this CMD file :)

Reply:
Thanks @rmuir I'll make the resolutions.